README.md

```diff
@@ -80,3 +80,6 @@ This project includes [medium-editor](https://github.com/yabwe/medium-editor/) b
 
 ## Contributing
 See CONTRIBUTING.md
+
+## Additional notes
```

Comment:
This seems to be really redundant to the calendar. Maybe we should stick to one name and code to handle when is which type of day, and dump the other?